doc/en/user/source/community/features-templating/directives.rst

```diff
@@ -0,0 +1,1211 @@
+.. _template-directives:
+
+Template Directives
+===================
+
+This part of the documentation is an introduction explaining the different template directives. 
+Examples will be provided for both Simple Features and Complex Features.
+The syntax of the directives varies slightly between XML based templates and JSON based templates.
+
+The examples will be provided mainly for GeoJSON and GML. Anyway the syntax defined for GeoJSON output, unless otherwise specified, will works as well for JSON-LD templates.
```

Comment:
```suggestion
The examples will be provided mainly for GeoJSON and GML. However the syntax defined for GeoJSON output, unless otherwise specified, is valid for JSON-LD templates.
```